### PR TITLE
[MIG] hr_holidays_leave_auto_approve: Migration to 14.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

**Review by flotho — APPROVED** [review]
functionally tested on runboat

File: hr_holidays_leave_auto_approve/tests/test_hr_holidays_leave_auto_approve.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# Copyright 2016-2019 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from odoo.tests.common import TransactionCase
+
+
+class TestHolidaysAutoValidate(TransactionCase):
+    def setUp(self):
```
Comment (ivantodorovich) [inline hr_holidays_leave_auto_approve/tests/test_hr_holidays_leave_auto_approve.py:10]: It'd be better to have this on `setUpClass` instead, to speed up tests.
Non blocking though

**Review by ivantodorovich — APPROVED** [review]
technically lg

**Review by i-vyshnevska — APPROVED** [review]

**Comment by houssine78** [comment]
@pedrobaeza seems that this PR need some love ;)

**Review by houssine78 — APPROVED** [review]

**Pull request merged**